core/domain/topic_services_test.py

```diff
@@ -230,13 +234,35 @@ def test_delete_subtopic_with_skill_ids(self):
             'cmd': topic_domain.CMD_DELETE_SUBTOPIC,
             'subtopic_id': self.subtopic_id
         })]
+        subtopic_page = topic_services.get_subtopic_page_by_id(
```

Comment:
Would it be better for the signature to be get_subtopic_page_by_id(topic_id, subtopic_id)?

That way, subtopic_page_id remains an "internal" concept, which simplifies things. Clients don't need to know about it.

Reply:
Yeah, done.